Mathlib/Algebra/MonoidAlgebra/Defs.lean

```diff
@@ -119,6 +119,8 @@ theorem single_apply {a a' : G} {b : k} [Decidable (a = a')] :
 @[simp]
 theorem single_eq_zero {a : G} {b : k} : single a b = 0 ↔ b = 0 := Finsupp.single_eq_zero
 
+theorem single_ne_zero {a : G} {b : k} : single a b ≠ 0 ↔ b ≠ 0 := Finsupp.single_ne_zero
```

Comment:
Do we already have the `AddMonoidAlgebra` version below?

Reply:
Seems like it was also missing!